src/app/components/tx-item.tsx

```diff
@@ -31,18 +31,39 @@ import {
 import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
 import { useFungibleTokenMetadata } from '@app/query/tokens/fungible-token-metadata.hook';
 import { pullContractIdFromIdentity } from '@app/common/utils';
+import { FiInfo } from 'react-icons/fi';
 
 type Tx = MempoolTransaction | Transaction;
+const pendingWaitingMessage =
+  'This transaction is waiting to be confirmed. Please be patient as this may take between a few minutes to a few hours.';
```

Comment:
```suggestion
  'This transaction is waiting to be confirmed. Depending on network congestion, this may take anywhere from a few minutes, to a couple of hours.';
```

Not sure we should directly ask for patience, he's another phrasing suggestion.